packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -1,60 +1,44 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-
 %global gem_name foreman_rh_cloud
 %global plugin_name rh_cloud
-%global foreman_min_version 2.5
+%global foreman_min_version 3.4
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 5.0.39
-Release: 2%{?foremandist}%{?dist}
+Name: rubygem-%{gem_name}
+Version: 6.0.42
+Release: 1%{?foremandist}%{?dist}
 Summary: Connects Foreman with Red Hat Cloud services
-Group: Applications/Systems
 License: GPLv3
 URL: https://github.com/theforeman/foreman_rh_cloud
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Autoreq: 0
-
-Obsoletes: %{?scl_prefix}rubygem-redhat_access
-Obsoletes: %{?scl_prefix}rubygem-redhat_access_lib
-
-Obsoletes: %{?scl_prefix}rubygem-foreman_inventory_upload
-Obsoletes: %{?scl_prefix}rubygem-foreman_inventory_upload-doc
-
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(katello)
-Requires: %{?scl_prefix}rubygem(foreman_ansible)
-Requires: %{?scl_prefix}rubygem(foreman-tasks)
+Requires: ruby
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
-BuildRequires: %{?scl_prefix}rubygem(katello)
-BuildRequires: %{?scl_prefix}rubygem(foreman_ansible)
-BuildRequires: %{?scl_prefix}rubygem(foreman-tasks)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby
+BuildRequires: ruby
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 Provides: foreman-plugin-%{plugin_name} = %{version}
+BuildRequires: rubygem(katello)
+BuildRequires: rubygem(foreman_ansible)
+BuildRequires: rubygem(foreman-tasks)
 # end specfile generated dependencies
 
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
-BuildRequires: %{?scl_prefix}npm(@babel/core) < 7.8.0
-BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) >= 2.5.0
-BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0
-BuildRequires: %{?scl_prefix}npm(jed) >= 1.1.1
-BuildRequires: %{?scl_prefix}npm(jed) < 1.2.0
+BuildRequires: npm(@babel/core) >= 7.7.0
+BuildRequires: npm(@babel/core) < 7.8.0
+BuildRequires: npm(@theforeman/builder) >= 8.16.0
+BuildRequires: npm(jed) >= 1.1.1
+BuildRequires: npm(jed) < 1.2.0
 # end package.json devDependencies BuildRequires
 
+# additional dependencies
+BuildRequires: npm(@redhat-cloud-services/frontend-components) >= 2.5.0
+BuildRequires: npm(@redhat-cloud-services/frontend-components) < 3.0.0
```

Comment:
If these are needed, why aren't they in `package.json`? It now looks like you're installing them in CI after you run `npm install`, but isn't that missing the point of a package manager? I'd consider this a bug.

Reply:
They are served by foreman-js, but for some reason we had to install them directly.
@ron-lavi do you know what's the status here for the latest Foreman?

Reply:
Given the build failed on it, perhaps the lower bound of foreman-js is not good enough? It may depend on https://github.com/theforeman/foreman-js/commit/8fe8afedecb8f02d8e1c61b1c485d92fda34bae3 which is part of `@theforeman/vendor` 10.1.1 but in packaging we have 10.1.0.

This is also why I hate that all plugins simply specify `>= 0` rather than a proper lower bound. We'd catch things more easily.